pipelines/batch/src/main/java/org/openmrs/analytics/FetchSearchPageFn.java

```diff
@@ -165,7 +165,13 @@ public void setup() throws SQLException, PropertyVetoException {
             sinkPath, sinkUsername, sinkPassword, fhirContext.getRestfulClientFactory());
     openmrsUtil = new OpenmrsUtil(sourceUrl, sourceUser, sourcePw, fhirContext);
     fhirSearchUtil = new FhirSearchUtil(openmrsUtil);
-    parquetUtil = new ParquetUtil(parquetFile, secondsToFlush, rowGroupSize, stageIdentifier + "_");
+    parquetUtil =
+        new ParquetUtil(
+            fhirContext.getVersion().getVersion(),
```

Comment:
what do we get by calling `getVersion()` twice that we don't get from calling it once?

Reply:
The first returns an `IFhirVersion` the second a `FhirVersionEnum`.